app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
```

Comment:
It is better to use `self__class__.people[name]`

Reply:
@HStakhiv @pavlo-kapitanets 
No, you do unnecessary actions using `self.__class__`, and readability is much worse. To access class attributes we should use the class name. We can also use `cls`, but only when it's a classmethod.

Reply:
Isn't that "hard code"? And I saw an example of such solution in one of the mentor's solutions on the platform

Reply:
In this task `Person.people[self.name] = self` is better because you are not going to inherit this class.
> Explicit is better than implicit.